src/analysis/retail/rogue/outlaw/modules/core/FinisherUse.tsx

```diff
@@ -0,0 +1,82 @@
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { CastEvent } from 'parser/core/Events';
+import { BadColor, GoodColor } from 'interface/guide';
+import { ResourceLink} from 'interface';
+import DonutChart from 'parser/ui/DonutChart';
+import Statistic from 'parser/ui/Statistic';
+import { STATISTIC_ORDER } from 'parser/ui/StatisticBox';
+import RESOURCE_TYPES from 'game/RESOURCE_TYPES';
+import getResourceSpent from 'parser/core/getResourceSpent';
+import {
+  FINISHERS,
+  getMaxComboPoints,
+} from '../../constants';
+import Finishers from '../features/Finishers';
+
+export default class FinisherUse extends Analyzer {
```

Comment:
This isn't a thing for this PR, but this indicates to me that we should probably have a shared version of this class somewhere.

Reply:
Yep, a lot of my first classes were mostly copy paste with slight changes of your assassination stuff, so having some of that shared would probably be a good idea.